packages/react-magma-dom/src/components/Pagination/SimplePagination.test.js

```diff
@@ -0,0 +1,290 @@
+      <Pagination type={PaginationType.simple} count={4} hidePreviousButton />
+    );
+    let previousButton = queryByText('Previous Page');
+
+    expect(previousButton).not.toBeInTheDocument();
+  });
+
+  it('Should hide the next button when the prop hideNextButton is used', () => {
+    const { queryByText } = render(
+      <Pagination type={PaginationType.simple} count={4} hideNextButton />
+    );
+    let nextButton = queryByText('Next Page');
+
+    expect(nextButton).not.toBeInTheDocument();
+  });
+
+  describe('Disabled', () => {
+    it('Should render disabled navigation icons, text, and select', () => {
+      const { getByText, getByLabelText } = render(
+        <Pagination type={PaginationType.simple} count={4} disabled />
+      );
+      const previousButton = getByLabelText('Previous Page');
+      const nextButton = getByLabelText('Next Page');
+
+      const selected = getByText('2').parentElement;
+
+      expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+      expect(selected).toHaveStyleRule(
+        'color',
+        transparentize(0.4, magma.colors.neutral500)
+      );
+      expect(selected).toHaveAttribute('disabled');
+
+      expect(previousButton).toHaveStyleRule(
+        'color',
+        transparentize(0.4, magma.colors.neutral500)
+      );
+      expect(nextButton).toHaveStyleRule(
+        'color',
+        transparentize(0.4, magma.colors.neutral500)
+      );
+      expect(previousButton).toHaveAttribute('disabled');
+      expect(nextButton).toHaveAttribute('disabled');
+    });
+
+    it('Should render inverse disabled navigation icons, text, and select', () => {
+      const { getByText, getByLabelText } = render(
+        <Pagination type={PaginationType.simple} count={4} isInverse disabled />
+      );
+      const previousButton = getByLabelText('Previous Page');
+      const nextButton = getByLabelText('Next Page');
+
+      const selected = getByText('2').parentElement;
+
+      expect(selected).toHaveStyleRule('cursor', 'not-allowed');
+
+      expect(selected).toHaveStyleRule(
+        'color',
+        transparentize(0.6, magma.colors.neutral100)
+      );
+      expect(selected).toHaveAttribute('disabled');
+
+      expect(previousButton).toHaveStyleRule(
+        'color',
+        transparentize(0.7, magma.colors.neutral100)
+      );
+      expect(nextButton).toHaveStyleRule(
+        'color',
+        transparentize(0.7, magma.colors.neutral100)
+      );
+      expect(previousButton).toHaveAttribute('disabled');
+      expect(nextButton).toHaveAttribute('disabled');
+    });
+
+    it('Should disable the previous button on the first page', () => {
+      const { getByLabelText } = render(
+        <Pagination type={PaginationType.simple} count={4} defaultPage={1} />
+      );
+      const previousButton = getByLabelText('Previous Page');
+
+      expect(previousButton).toHaveStyleRule(
+        'color',
+        transparentize(0.4, magma.colors.neutral500)
+      );
```

Comment:
Instead of checking the colors, we could expect the button `.toBeDisabled()`. The color checking should happen in the Button tests because that's part of the button component. The logic in this component just cares that the element became disabled. 

_(same feedback below for next button)_